index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please add a descriptive PR. 
**_Pull request should include the list of changes that you have made in this branch_**

Reply:
thanks for your quick review